src/email_validator.gs.js

```diff
@@ -129,6 +130,19 @@ function cleanUpEmailList(emails, options) {
   return validFields;
 }
 
+/**
+ * Clean up a single email address by removing white-space characters
+ *
+ * @param {string} potentially dirty email address
+ *
+ * @return {string} cleaned-up email address ("Hervé.Du Chène@gmail.com" --> "Example Foo") or null
+ */
+function cleanUpSingleAddress(email) {
+  return cleanUpEmailList(email.replace(/\s/g, ''), {
```

Comment:
I would move the space removing code to the main clean up function, so the whole cleanup process is consistent ?

Reply:
Done